src/App.jsx

```diff
@@ -14,70 +15,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export default function App() {
+  const jam = goods.find(e => e === 'Jam');
+  const [item, setItem] = useState(jam);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function itemPick(elem) {
+    setItem(elem);
+  }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  function itemClear() {
+    setItem('');
+  }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {item === '' ? (
```

Comment:
there is no need to need write this check

you can write simplier  `{selectedItem ? (`

how is it work?
this value will be automatically converted to boolean...so if its an `empty string, 0 or null ` then it will be false, if we have some text then it would be true